compiler/onecc-docker/onecc-docker

```diff
@@ -46,9 +47,26 @@ def main():
     script_path = os.path.dirname(os.path.realpath(__file__))
     dockerfile_path = os.path.join(script_path, 'docker')
 
+    onecc_docker_usage = 'onecc-docker [-h] [-t TOKEN] [COMMAND <args>]'
+    onecc_docker_desc = 'Run onecc via docker'
+    parser = argparse.ArgumentParser(
+        usage=onecc_docker_usage, description=onecc_docker_desc)
+    parser.add_argument(
+        "-t",
+        "--token",
+        help=
+        "Token for authentication to GitHub. This is a workaround for Rate limit exceeded error"
+    )
+
+    args, onecc_arguments = parser.parse_known_args()
+    authorization_token = args.token
+
     LATEST_URL = "https://api.github.com/repos/Samsung/ONE/releases/latest"
     try:
-        response = requests.get(LATEST_URL)
+        headers = {}
+        if authorization_token:
+            headers = {"Authorization": "Bearer {}".format(authorization_token)}
```

Comment:
It would be better to take it out of the `try` statement except for the `requests` function.

Reply:
Thank you for advice! I took out `header` related part from `try` statement.